### ISSUE TITLE
grpc_testing module is underdocumented

### ISSUE THREAD

**Issue by neliel123** [grpc/grpc#19150]
### What version of gRPC and what language are you using?
 1.11
 
### What operating system (Linux, Windows,...) and version?
 Linux
 
### What runtime / compiler are you using (e.g. python version or version of gcc)
 Python
 
### What did you do?
If possible, provide a recipe for reproducing the error. Try being specific and include code snippets if helpful.

Other than this link https://grpc.github.io/grpc/python/grpc_testing.html, I don't see anything that describes how do I test my grpc server.

Is the way to test GRPC services written in python is by executing a full server-client test?
I was looking to mock some of my server code and execute unit test. 

### What did you see instead?
 
Make sure you include information that can help us debug (full error message, exception listing, stack trace, logs).

See [TROUBLESHOOTING.md](https://github.com/grpc/grpc/blob/master/TROUBLESHOOTING.md) for how to diagnose problems better.

**Comment by gnossen** [comment]
@neliel123 If you're looking for a way to mock out gRPC servicers, then `grpc_testing` is definitely the way to go. However, as you've noticed, there isn't currently a whole lot of documentation on the module. This is perhaps a symptom of that fact that the majority of the tests we write for this repo care very much about the lower-level networking aspects of the system and, as such, are almost always a full client-server test.

With that said, even without the documentation, `grpc_testing` should meet your needs if you're willing to wade through the code. We'll add documenting the module to our to-do list, but I'm not sure how much priority it will get. In the meantime, we're absolutely open to accepting PRs for documentation.

**Comment by neliel123** [comment]
Thanks for the inputs... I am not sure if this is the grpc_testing you are mentioning [grpcio_testing](https://github.com/grpc/grpc/tree/master/src/python/grpcio_testing)

**Comment by gnossen** [comment]
That's right. The pip package is called `grpcio_testing`, but the module is called `grpc_testing`.

**Comment by gnossen** [comment]
@yashykt Does closing this issue mean that we've determined that keeping track of this technical debt is no longer valuable? I for one think that this is still a valid goal. We just haven't had enough hands to handle this since May.

**Comment by yashykt** [comment]
@gnossen I closed this because it was marked as kind/question and from a quick read it felt like that the question was answered. On re-reading, it looks like it was more a enhancement request. Reopened it

**Comment by gnossen** [comment]
You're right. When I updated the title, I didn't update the labelling. That's my bad.
